packages/core/src/components/ui/Button/ButtonSignIn/ButtonSignIn.tsx

```diff
@@ -15,10 +15,10 @@ const ButtonSignIn = () => {
       }
       className="text__title-mini"
       variant="tertiary"
-      icon={<Icon name="User" width={18} height={18} weight="bold" />}
+      icon={<Icon name={icon} width={18} height={18} weight="bold" />}
       iconPosition="left"
     >
-      {person?.id ? 'My Account' : 'Sign In'}
+      {person?.id ? 'My Account' : label}
```

Comment:
I wonder if we should extract the `My Account` from CMS as well 🤔 . WDYT?

Reply:
Humm I'm not sure, because the CMS won't have the state context for this button. But maybe it would be good to have a specific type for this label when user session exists.

Reply:
Added a property to handle that info ✅